pnpm-lock.yaml

```diff
@@ -15669,7 +15728,7 @@ snapshots:
       semver: 7.7.2
       source-map: 0.7.4
       typescript: 5.8.3
-      webpack: 5.99.8
+      webpack: 5.99.8(esbuild@0.25.5)
```

Comment:
I am a bit curious what happened with the webpack change. I just updated my global Node and pnpm